example-site/components/templates/MultiPageFormSuccess.tsx

```diff
@@ -28,7 +37,7 @@ export function MultiPageFormSuccess() {
 							pretext="Title of multi-page form"
 							title="Multi-page form title (H1)"
 						/>
-						<PageAlert tone="success">
+						<PageAlert ref={successPageAlertRef} tabIndex={-1} tone="success">
```

Comment:
I consistently cannot get the banner to focus for Multi-step only (single page is fine)

Reply:
@nathsimpson I'm really sorry, the link in my description was to the live site and not the PR preview 🤭